### ISSUE TITLE
Support RHEL7

### ISSUE THREAD

**Issue by sdbrockme** [convergeone/freeztp#59]
Thanks for creating this awesome project.  I took a look at the python code and there is a check to make sure the OS is one of the 3 supported OS's.  I'm in a corporate environment and we must run RHEL for all linux instances.  Is there any way to build in support for RHEL?

**Comment by sdbrockme** [comment]
Here's what I get when running the platform.linux_distribution() command:

[v00utlrhl401 freeztp]$ python -c 'import platform;print(platform.linux_distribution())'
('Red Hat Enterprise Linux Server', '7.7', 'Maipo')

**Comment by PackeTsar** [comment]
what do you get when you run the command `ztp`

**Comment by PackeTsar** [comment]
Or if you have not installed it: `python ztp.py`

**Comment by sdbrockme** [comment]
I have not installed yet.  Looking at your installer I was concerned it would error out. based on the definition for _dist_detect:

I unfortunately can't run the installer and have to request someone else run it.  Just attempting to make sure the installer will work as the request process is a PIA. 

I'm going to go ahead and request it to be installed. I'll report back here.

Thanks
Seth

**Comment by sdbrockme** [comment]
As I thought, we had to edit the script to return "centos" for the linux distro.   T

              distlist = platform.linux_distribution()
                return "centos"


The script had some errors when running and failed to fully install. I've attached the output. 
[FreeZtp fail.txt](https://github.com/PackeTsar/freeztp/files/3962542/FreeZtp.fail.txt)


I've found something.  Looks like our proxy is returning an HTML file instead of the TFTPy project file. 
Working on getting through the proxy
